hardhat.config.ts

```diff
@@ -63,7 +65,7 @@ const config: HardhatUserConfig = {
     },
     arbitrum: {
       url: process.env.ARBITRUM_MAINNET_URL,
-      accounts: [`${process.env.ARBITRUM_MAINNET_PK}`],
+      accounts: [`${privateKey}`],
```

Comment:
Recommend reverting this to not wreck people's repo setups

Reply:
Reverting.